lib/text/bugs

```diff
@@ -198,3 +198,4 @@ Aellyn   (Jan  1) [31102] mortally wounded characters can counterstrike and hit
 Shadow   (Jul 10) [ 3474] I remember being able to get a snowflake downloaded from here in JNL to unlock the door to the next room but the files don't seem to be spawning
 Armitage (Sep 23) [ 2961] underflow in points-based creation if you play a race that uses more than what you put into attributes. I made a troll with 6 in attributes and had 65k training points (I deleted and remade with the 11 attribute points needed but most people wouldn't, probably)
 Lucien   (Jul  5) [35501] release all gave no reply when I used it
+Lucien   (Mar 13) [60611] equipment flag from aff_thermo etc stays on you from cyberware/bioware even after it's sold
```

Comment:
Need to fix this, then remove this line from the bugs file-- we don't want to edit this file if possible.